subtests/docker_cli/run/run.py

```diff
@@ -73,6 +73,7 @@ def run_once(self):
     def postprocess(self):
         super(run_base, self).postprocess()  # Prints out basic info
         dockercmd = self.sub_stuff['dkrcmd']
+        self.logdebug(dockercmd)
```

Comment:
I'm having a hard time with this. My sense is that it's overkill: The docker command is already logged a few lines earlier; we don't really need to see exit status or stderr because those are already checked for and displayed on error. What I was asking for is just the stdout from 'yum update', because I keep thinking there will be cases where the test passes without actually testing secrets. Maybe I'm just being too paranoid.

The added verbosity, in an already WAY too verbose debug log (the run test), bothers me more than a merely hypothetical failure. I take back my request: let's just skip logging, and deal with problems if they arise.